### PR TITLE
doc: K8s tutorial

### PR DESCRIPTION
**Pull request by hydratim**
#### Summary

Adds Kubernetes CronJob tutorial to the deployment documentation.

### CODE REVIEW

File: website/pages/docs/deployment/kubernetes.mdx [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+image: https://i3.ytimg.com/vi/3dJVj6JjHqY/maxresdefault.jpg
+video: https://youtu.be/3dJVj6JjHqY
+---
+
+import { Callout } from 'nextra-theme-docs';
+
+# Deploying CloudQuery using Kubernetes CronJobs
+
+In this tutorial we will set up a Kubernetes CronJob to run CloudQuery Sync on a regular schedule.
+Due to its flexibility and standardization across all cloud providers, Kubernetes is the #1 way that DevOps & Platform Engineers prefer to deploy workloads and microservices.
```
Comment (yevgenypats) [inline website/pages/docs/deployment/kubernetes.mdx:15]: Maybe keep `Kubernetes is the #1...` out? and keep it a bit more opinionated.

**Review by yevgenypats — APPROVED** [review]
Looks great! had one nit.

File: website/pages/docs/deployment/kubernetes.mdx (hunk 1 above)
Comment (hydratim) [reply website/pages/docs/deployment/kubernetes.mdx:15]: I feel that the search numbers support the claim, but I'm happy to change it.
Would this work better for you?
```suggestion
Due to its flexibility and standardization across cloud providers, Kubernetes is commonly used by DevOps & Platform Engineers when deploying workloads and microservices.
```

**Comment by songant** [comment]
Does cloudquery deployed in kubernetes support writing the synchronized data to S3? In addition, when synchronizing aws or gcp cloud resource data, the credentials file is directly written to the default path in the pod (such as $root/.aws/credentials )

**Pull request merged**